girder/girder_large_image/web_client/vue/components/CompositeLayers.vue

```diff
@@ -68,63 +68,65 @@ export default {
                 this.toggleColorPicker(undefined);
             }
         },
-        updateChannelColor(channel, swatch) {
-            this.compositeChannelInfo[channel].falseColor = swatch.hex;
+        updateLayerColor(layer, swatch) {
+            this.compositeLayerInfo[layer].falseColor = swatch.hex;
         },
-        updateChannelMin(event, channel) {
+        updateLayerMin(event, layer) {
             const newVal = event.target.value;
             const newMinVal = parseFloat(newVal);
-            this.compositeChannelInfo[channel].min = newMinVal;
+            this.compositeLayerInfo[layer].min = newMinVal;
             this.updateStyle();
         },
-        updateChannelMax(event, channel) {
+        updateLayerMax(event, layer) {
             const newVal = event.target.valueAsNumber;
             const newMaxVal = parseFloat(newVal);
-            this.compositeChannelInfo[channel].max = newMaxVal;
+            this.compositeLayerInfo[layer].max = newMaxVal;
             this.updateStyle();
         },
-        updateActiveChannels() {
-            this.channels.forEach((channel) => {
-                this.compositeChannelInfo[channel].enabled = this.enabledChannels.includes(channel);
+        updateActiveLayers() {
+            this.layers.forEach((layer) => {
+                this.compositeLayerInfo[layer].enabled = this.enabledLayers.includes(layer);
             })
             this.updateStyle();
         },
         updateStyle() {
-            const activeChannels = Object.values(
-                this.compositeChannelInfo
-            ).filter((channel) => channel.enabled);
+            const activeLayers = Object.values(
+                this.compositeLayerInfo
+            ).filter((layer) => layer.enabled);
             const styleArray = []
-            activeChannels.forEach((channel) => {
-                const styleEntry = {
-                    frameDelta: channel.number,
-                };
-                if (channel.falseColor) {
-                    styleEntry['palette'] = channel.falseColor;
+            activeLayers.forEach((layer) => {
+                // TODO: what should each style entry look like for band compositing?
```

Comment:
Instead of a `"framedelta"` parameters (technically it could be in addition to rather than instead of), you use a `"band"` parameter.  Its value is either the unique interpretation string or the 1-based band number.  See https://girder.github.io/large_image/tilesource_options.html#style.

Reply:
Thanks. Changed in [1ba2522](https://github.com/girder/large_image/pull/1086/commits/1ba252219be40e7cc804ad437c32977a10bf958c)